### PR TITLE
Change pending label to 'Minimum balance for payout is X currency'. C…

### PR DESCRIPTION
**Pull request by yachtcaptain23**
…loses #2387

### CODE REVIEW

File: config/locales/ja.yml [hunk 1]
```diff
@@ -554,6 +554,7 @@ ja:
       balance_pending: 保留中の支払い
       next_deposit_date: "次の入金日："
       payout_in_progress: "進行中の支払い"
+      payout_minimum_balance: "Minimum balance for payout is %{amount} %{currency}"
```
Comment (corymcdonald) [inline config/locales/ja.yml]: Reminder to get the japanese locale before merge. :)

File: app/models/payout_report.rb [hunk 2]
```diff
@@ -4,6 +4,9 @@ class PayoutReport < ApplicationRecord
 
   LEGACY_PAYOUT_REPORT_TRANSITION_DATE = "2018-12-01 09:14:58 -0800".freeze
 
+  MINIMUM_BALANCE_AMOUNT = 5
+  BAT = 'bat'.freeze
```
Comment (corymcdonald) [inline app/models/payout_report.rb]: Is this being used anywhere?

Comment (yachtcaptain23) [inline app/models/payout_report.rb]:
```suggestion
```

File: config/locales/ja.yml (hunk 1 above)
Comment (yachtcaptain23) [reply config/locales/ja.yml]: I'm aware :)

Comment (yachtcaptain23) [inline config/locales/ja.yml]:
```suggestion
      payout_minimum_balance: "最低出金金額は %{amount} %{currency}です"
```

Comment (corymcdonald) [reply config/locales/ja.yml]: Do you have the locales?

Comment (yachtcaptain23) [reply config/locales/ja.yml]:
```suggestion
      payout_minimum_balance: "最低出金金額は %{amount} %{currency}です"
```

**Pull request merged**